src/utils.ts

```diff
@@ -1,67 +1,66 @@
 import { Validations, Value, Values } from './interfaces';
 
-export function arraysDiffer(a: unknown[], b: unknown[]) {
-  let isDifferent = false;
-  if (a.length !== b.length) {
-    isDifferent = true;
-  } else {
-    a.forEach((item, index) => {
-      if (!this.isSame(item, b[index])) {
-        isDifferent = true;
-      }
-    }, this);
-  }
-  return isDifferent;
+export function isArray(value: unknown): value is unknown[] {
+  return Array.isArray(value);
```

Comment:
Why do you need this method if there's already a built in `Array.isArray()` method?

Reply:
Hm, conformity and brevity? `isArray` is shorter than `Array.isArray` and looks cleaner below next to the other `is*` functions. Though you're right, it's not as justifiable as the others.

I could also see us adding something to this function to make it safer like Lodash does: https://github.com/lodash/lodash/blob/master/isArrayLike.js#L27